binary-package.md

```diff
@@ -0,0 +1,76 @@
+---
+title: TiDB Installation Packages
+summary: Learn about TiDB installation packages and the specific components included.
+---
+
+# TiDB Installation Packages
+
+Before [deploying TiUP offline](/production-deployment-using-tiup.md#deploy-tiup-offline), you need to download the binary packages of TiDB at the [official download page](https://en.pingcap.com/download/).
+
+TiDB provides two binary packages: `TiDB-community-server` and `TiDB-community-toolkit`
+
+The `TiDB-community-server` package contains the following contents.
+
+| Content | Change History |
```

Comment:
```suggestion
| Content | Change history |
```
See https://developers.google.com/style/capitalization#capitalization-for-tables-in-text